package.json

```diff
@@ -78,7 +78,7 @@
     "ts-jest": "^25.4.0",
     "ts-node": "^8.8.1",
     "typedoc": "^0.17.6",
-    "typedoc-plugin-lerna-packages": "^0.3.0",
+    "typedoc-plugin-lerna-packages": "0.3.0",
```

Comment:
Use the latest version `0.3.1` will introduce a bug that the readme file will be gone.